schema/src/lib.rs

```diff
@@ -72,10 +69,10 @@ pub enum Metadata {
     Array(ArrayMeta),
     /// Vector with type
     Vec(String),
+    /// Vector with type where elements are ordered
+    SortedVec(String),
```

Comment:
in my opinion this variant structure should also include comparison function as a 2nd parameter